src/views/List.jsx

```diff
@@ -23,9 +25,23 @@ export function List({ data }) {
 			<p>
 				Hello from the <code>/list</code> page!
 			</p>
+			{data.length ? (
+				<p>
+					Add additional items on the
+					<button
+						aria-label="Navigate to Manage List page"
```

Comment:
It might make more sense to remove this aria-label and instead just use the default name (which will be the contents displayed inside the button, in this case "/Manage List"), so that the screen reader announcements will make a bit more sense as users navigate through the page. I'll post a video in Slack to explain what I mean.

Reply:
https://the-collab-lab.slack.com/archives/C07FYKDSUN8/p1725706029459859

Reply:
Oh and actually, if it's navigating to another page, you may want to use a link `<a>` instead of a button. The general rule of thumb I've heard is that buttons are for actions within a page, links are for navigating to a different page.